Doc/library/stdtypes.rst

```diff
@@ -1220,7 +1220,7 @@ accepts integers that meet the value restriction ``0 <= x <= 255``).
 Notes:
 
 (1)
-   *t* must have the same length as the slice it is replacing.
+   If *k* != 1, *t* must have the same length as the slice it is replacing.
```

Comment:
The majority of literals in this file are marked up with double backquotes, similar to L1226.

```suggestion
   If *k* != ``1``, *t* must have the same length as the slice it is replacing.
```

I would consider to spell it out, though:

```suggestion
   If *k* is not equal to ``1``, *t* must have the same length as the slice it is replacing.
```